### DISCUSSION TITLE
Thoughts on this action / organizing of repo!

### DISCUSSION THREAD

**Discussion by MarketingPip** [MarketingPipeline/Markdown-Tag#8] [Q&A]
@DarkenLM  @shaunbharat - wanting to hear your thoughts on adding this action to the repo. 

https://github.com/marketplace/actions/todo-to-issue **(This for sure I am thinking)** 

As well implementing all tasks to-do.md lists etc. into issues. (This I am not so sure of)

Kinda hesitant as when anyone see's the word issues and a large amount beside it, it usually tend's to scare them. Which if we had a lot of to-do's in list > would result in ton of open issues.

Looking to hear your thoughts / input on how we should go about organizing the to-do list (maybe add a section in discussion etc?).

**Comment by shaunbharat** [comment]
I think we could add the to-do items to a pinned discussion in a [task list](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists). But using issues is useful, since if the to-do item is finished, we could reference the issue in the PR.

About a large issue number scaring people -  having the to-do list in a discussion might be the best solution. Though I don't think it might be a huge problem.

**Reply by MarketingPip** [reply]
Thank you @shaunbharat - I think  we will go ahead and add a to-do in discussion list to keep things clean! Hoping you're having a great weekend as well! 

@DarkenLM any thoughts on using this action in the script? We could change issue tag marker obviously for our own issues instead of the current one.

**Reply by DarkenLM** [reply]
In my opinion, a repo with a lot of issues which are clearly being responded and acted upon is way more valuable than no issues at all. When I look at a github project, the issue panel is the next thing I see after the README and part of the source code, and if there are lots of issues which were not ignored, but are being worked on, I take it as a sign the code is well maintained.

**Reply by shaunbharat** [reply]
I was going to mention this as well - it does show activity and interaction.

**Comment by MarketingPip** [comment]
@DarkenLM @shaunbharat  - Thanks for quick reply! If that's the case I think we will go ahead and implement this action in when we make the pull request when thing's are ready! 

I will ask you both to look into the this (if you already have not) - [GitHub Action](https://github.com/marketplace/actions/todo-to-issue)

And start marking code accordingly. I think we should mark the tag / and modify the action to reflect `##Markdown-Tag TO-DO`

As well it would be great if comments could be provided throughout the script for any other developers / or future developers that might need to go through this. I was thinking we could base our comments off Taylor Otwell's amazing comment structure / template. 

Comment example from Laravel - 

```

/*
|--------------------------------------------------------------------------
| Register The Auto Loader
|--------------------------------------------------------------------------
|
| Composer provides a convenient, automatically generated class loader for
| this application. We just need to utilize it! We'll simply require it
| into the script here so we don't need to manually load our classes.
|
*/
```
